Index: src/test.bat
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>javac .\\hust\\soict\\hedspi\\aims\\test\\cart\\TestCart.java\r\njava hust/soict/hedspi/aims/test/cart/TestCart
===================================================================
diff --git a/src/test.bat b/src/test.bat
--- a/src/test.bat	
+++ b/src/test.bat	
@@ -1,2 +1,0 @@
-javac .\hust\soict\hedspi\aims\test\cart\TestCart.java
-java hust/soict/hedspi/aims/test/cart/TestCart
\ No newline at end of file
Index: src/testtostring.bat
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>javac hust/soict/hedspi/test/cart/TestToString.java\r\njava hust/soict/hedspi/test/cart/TestToString
===================================================================
diff --git a/src/testtostring.bat b/src/testtostring.bat
--- a/src/testtostring.bat	
+++ b/src/testtostring.bat	
@@ -1,2 +1,0 @@
-javac hust/soict/hedspi/test/cart/TestToString.java
-java hust/soict/hedspi/test/cart/TestToString
\ No newline at end of file
Index: src/store.bat
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>javac .\\hust\\soict\\hedspi\\test\\store\\StoreTest.java\r\njava hust/soict/hedspi/test/store/StoreTest
===================================================================
diff --git a/src/store.bat b/src/store.bat
--- a/src/store.bat	
+++ b/src/store.bat	
@@ -1,2 +1,0 @@
-javac .\hust\soict\hedspi\test\store\StoreTest.java
-java hust/soict/hedspi/test/store/StoreTest
\ No newline at end of file
Index: src/testsort.bat
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>javac hust/soict/hedspi/test/cart/TestSortCart.java\r\njava hust/soict/hedspi/test/cart/TestSortCart
===================================================================
diff --git a/src/testsort.bat b/src/testsort.bat
--- a/src/testsort.bat	
+++ b/src/testsort.bat	
@@ -1,2 +1,0 @@
-javac hust/soict/hedspi/test/cart/TestSortCart.java
-java hust/soict/hedspi/test/cart/TestSortCart
\ No newline at end of file
